pmm/pmm2-server-release.groovy

```diff
@@ -172,22 +172,27 @@ pipeline {
                     sg docker -c "
                         set -ex
                         docker pull \${DOCKER_VERSION}
+                        if [ \${TOP_VER} -gt 1 ]; then
```

Comment:
Why we need that condition in PMM 2 pipeline?

Reply:
Lack of my knowledge ) It was there before, I thought we need for some reason...

Reply:
@EvgeniyPatlan @hors Do we still need those checks in this file?

Reply:
OK, removed them assuming this pipeline doesn't get called from another one.